packages/d3fc-webgl/src/series/glLine.js

```diff
@@ -28,16 +29,15 @@ export default () => {
     const yPreviousValueAttribute = elementConstantAttributeBuilder().value(
         (data, element) => data[Math.max(element - 1, 0)]
     );
-    const cornerAttribute = projectedAttributeBuilder()
+    const cornerAttribute = vertexConstantAttributeBuilder()
         .size(2)
         .data([
             [-1, -1],
             [1, -1],
             [-1, 1],
             [1, 1]
-        ])
-        .value((data, element, vertex, component) => data[vertex][component]);
-    const definedAttribute = projectedAttributeBuilder().value(
+        ]);
+    const definedAttribute = elementConstantAttributeBuilder().value(
```

Comment:
Same question as above - both elementConstant and vertexConstant display the graph as intended but not sure of the 'invisible' implications of either one here.